VK/VK/Presentation/Profile/ViewControllers/ProfileCollectionViewController.swift

```diff
@@ -14,7 +14,7 @@ final class ProfileCollectionViewController: UICollectionViewController {
 
     // MARK: - Public Properties
 
-    var friend: Friend?
+    var friend: NewsFriends?
```

Comment:
```suggestion
    var friendNews: FriendNews?
```
это же не друзья а их новости называй правильно